packages/react/src/input/input.styles.ts

```diff
@@ -1,5 +1,6 @@
 import {styled, VariantProps} from "../theme/stitches.config";
 import {sharedVisuallyHidden} from "../theme/shared-css";
+import {cssFocusVisible} from "../theme/shared-css";
```

Comment:
Can you move this import to the above? There is already an import from 'theme/shared-css'